demo-output.yaml

```diff
@@ -27,11 +27,6 @@
       innerText: "\n      io.fabric8\n      kubernetes-client-api\n      6.0.0\n    "
       matchingXML: <groupId>io.fabric8</groupId><artifactId>kubernetes-client-api</artifactId><version>6.0.0</version>
   extras: []
-- ruleID: chain-pom-002
```

Comment:
I don't think anything I did should lead to this being excluded?

Reply:
with no incidents, I wonder if we are returning the wrong value in the chain?

Reply:
or if it really should not be matched